docs/index.asciidoc

```diff
@@ -25,6 +25,34 @@ This Logstash input plugin allows you to call an HTTP API, decode the output of
 send them on their merry way. The idea behind this plugins came from a need to read springboot
 metrics endpoint, instead of configuring jmx to monitor my java application memory/gc/ etc.
 
+[id="plugins-{type}s-{plugin}-ecs_metadata"]
```

Comment:
@kaisecheng Please consider moving the ECS info below the example. The example relates to the general use and operation of the plugin, and the current placement makes it looks like it's specific to ECS.